settings.gradle

```diff
@@ -23,7 +23,7 @@ dependencyResolutionManagement {
             library('mysql', 'com.mysql:mysql-connector-j:8.0.32')
             library('mapstruct', "org.mapstruct:mapstruct:$mapStructVersion")
             library('mapstructAnnotationProcessor', "org.mapstruct:mapstruct-processor:$mapStructVersion")
-            library('kafka', 'org.springframework.kafka:spring-kafka:3.0.5')
+            library('kafka', "org.springframework.kafka:spring-kafka:$springVersion")
```

Comment:
Please note for future that kafka itself is super complicated inside and pretty fragile, thus spring-kafka version bump should be performed super careful and distributed to developers after testing by devops/ite specialists. In case you decide to leave kafka version as variable, please do not update spring until we finish our project.